libcodechecker/libhandlers/store.py

```diff
@@ -182,44 +182,67 @@ def collect_file_hashes_from_plist(plist_file):
+                hash_to_file[content_hash] = f
+                file_to_hash[f] = content_hash
+                file_to_mtime[f] = util.get_last_mod_time(f)
 
             return True
         except Exception as ex:
             LOG.error('Parsing the plist failed: ' + str(ex))
 
-    with zipfile.ZipFile(zip_file, 'a', allowZip64=True) as zipf:
-        for input_path in inputs:
-            input_path = os.path.abspath(input_path)
+    plist_report_files = []
 
-            if not os.path.exists(input_path):
-                raise OSError(errno.ENOENT,
-                              "Input path does not exist", input_path)
+    changed_files = set()
+    for input_path in inputs:
+        input_path = os.path.abspath(input_path)
 
-            if os.path.isfile(input_path):
-                files = [input_path]
-            else:
-                _, _, files = next(os.walk(input_path), ([], [], []))
+        if not os.path.exists(input_path):
+            raise OSError(errno.ENOENT,
+                          "Input path does not exist", input_path)
 
-            for f in files:
-                plist_file = os.path.join(input_path, f)
-                if f.endswith(".plist"):
-                    if collect_file_hashes_from_plist(plist_file):
-                        LOG.debug(
-                            "Copying file '{0}' to ZIP assembly dir..."
-                            .format(plist_file))
-                        zipf.write(plist_file, os.path.join('reports', f))
-                    else:
-                        LOG.warning("Skipping '{0}' because it contains "
-                                    "a missing source file."
-                                    .format(plist_file))
-                elif f == 'metadata.json':
-                    zipf.write(plist_file, os.path.join('reports', f))
+        if os.path.isfile(input_path):
+            files = [input_path]
+        else:
+            _, _, files = next(os.walk(input_path), ([], [], []))
+
+        for f in files:
+            plist_file = os.path.join(input_path, f)
+            if f.endswith(".plist"):
+                if collect_file_hashes_from_plist(plist_file):
+                    LOG.debug(
+                        "Copying file '{0}' to ZIP assembly dir..."
+                        .format(plist_file))
+                    plist_report_files.append(os.path.join(input_path, f))
+                else:
+                    LOG.warning("Skipping '{0}' because it contains "
+                                "a missing source file."
+                                .format(plist_file))
+            elif f == 'metadata.json':
+                plist_report_files.append(os.path.join(input_path, f))
+
+            plist_mtime = util.get_last_mod_time(plist_file)
+
+            for k, v in file_to_mtime.items():
+                if v > plist_mtime:
+                    changed_files.add(k)
+
+    if changed_files:
+        LOG.info("Changed source files:")
```

Comment:
Use the same output here as at the parse command. E.g.:
```bash
[2018-02-12 17:02] - The following source file contents changed since the latest analysis:
 - /home/user/projects/simple/main.cpp
 - /home/user/projects/simple/main2.cpp
Please analyze your project again to update the reports!
```